XLS-47d-PriceOracles/README.md

```diff
@@ -0,0 +1,468 @@
+       "ledger_current_index" : 23,
+       "node" : {
+          "Flags" : 0,
+          "LastUpdateTime" : 743609014,
+          "LedgerEntryType" : "Oracle",
+          "Owner" : "rp847ow9WcPmnNpVHMQV5A4BF6vaL9Abm6",
+          "AssetClass" : "63757272656E6379",
+          "Provider": "70726F7669646572",
+          "PreviousTxnID" : "6F120537D0D212FEA6E11A0DCC5410AFCA95BD98D451D046832E6C4C4398164D",
+          "PreviousTxnLgrSeq" : 22,
+          "PriceDataSeries": [
+            {
+              "PriceData: {
+                "QuoteAsset" : {
+                   "currency" : "USD"
+                },
+                "BaseAsset" : {
+                   "currency" : "XRP"
+                },
+                "Scale" : 1,
+                "AssetPrice" : "740",
+              }
+            }
+          ],
+          "index" : "CF2C20122022DE908C4F521A96DC2C1E5EFFD1EFD47AA244E9EE9A442451162E"
+       },
+       "status" : "success",
+       "validated" : true
+    }
+
+#### Oracle Aggregation
+
+`get_aggregate_price` RPC calculates the aggregate price of the specified `PriceOracle`, and returns three types of price statistics - average, median, and trimmed median if `trim` parameter is included in the request.
+
+##### Input API fields
+
+|FieldName | Required? | JSON Type |
+|:---------|:-----------|:---------------|
+| `ledger_index` | | `string` or `number` (positive integer)|
+
+The ledger index of the max ledger to use, or a shortcut string to choose a ledger automatically.
+
+|FieldName | Required? | JSON Type |
+|:---------|:-----------|:---------------|
+| `ledger_hash` | | `string` |
+
+A 20-byte hex string for the max ledger version to use.
+
+|FieldName | Required? | JSON Type |
+|:---------|:-----------|:---------------|
+| `base_asset` | :heavy_check_mark: | `string` |
+
+`base_asset` is the asset to be priced.
+
+|FieldName | Required? | JSON Type |
+|:---------|:-----------|:---------------|
+| `quote_asset` | :heavy_check_mark: | `string` |
+
+`quote_asset` is the denomination in which the prices are expressed.
+
+|FieldName | Required? | JSON Type |
+|:---------|:-----------|:---------------|
+| `oracles` | :heavy_check_mark: | `array` |
+
+`oracles` is an array of `oracle` objects to aggregate over. `oracle` object has two fields:
+
+- |FieldName | Required? | JSON Type |
+  |:---------|:-----------|:---------------|
+  | `account` | :heavy_check_mark: | `string` |
+
+    `account` is the Oracle's account.
+
+- |FieldName | Required? | JSON Type |
+  |:---------|:-----------|:---------------|
+  | `oracle_sequence` | :heavy_check_mark: | number |
+
+  `oracle_sequence` is a unique identifier of the Price Oracle for the given Account.
+
+|FieldName | Required? | JSON Type |
+|:---------|:-----------|:---------------|
+| `trim` | | `number` |
+
+`trim` is the percentage of outliers to trim. Valid trim range is 1-25. If this parameter is included then the API returns trimmed median in addition to the average and median.
```

Comment:
"trimmed median" ? I think the term is "trimmed mean". Also, would suggest using "mean" instead of average through because it is more precise.

Reply:
Thanks. It's a typo. The implementation has the trimmed average. I'm ok with using `mean` instead.